src/NServiceBus.Persistence.AzureStorage/Timeout/TimeoutLogic/TimeoutPersister.cs

```diff
@@ -164,9 +166,9 @@ public TimeoutPersister(string timeoutConnectionString, string timeoutDataTableN
                     .Where(
                         TableQuery.CombineFilters(
                             TableQuery.CombineFilters(
-                                TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.NotEqual, now.ToString(partitionKeyScope)),
+                                TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.LessThanOrEqual, now.ToString(partitionKeyScope)),
```

Comment:
For the readability, since this code is changing, would it improve readability if to swap the two conditions?
Instead of reading it as  
`PK <= now AND PK >= lastSuccessfulRead`
to have
`PK >= lastSuccessfulRead AND PK <= now`